release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -388,6 +388,15 @@ For more information about deployment specifications for MetalLB, see xref:../ne
 [id="ocp-4-12-olm"]
 === Operator lifecycle
 
+[id="ocp-4-12-platform-operators"]
+==== Platform Operators (Technology Preview)
+
+Starting in {product-title} 4.12, Operator Lifecycle Manager (OLM) introduces the _platform Operator_ type as a Technology Preview feature. The platform Operator mechanism relies on resources from the RukPak component, also introduced in {product-title} 4.12, to source and manage content.
+
+A platform Operator is an OLM-based Operator that can be installed during or after an {product-title} cluster's Day 0 operations and participates in the cluster's lifecycle. As a cluster administrator, you can use platform Operators to further customize your OpenShift Container Platform installation to meet your requirements and use cases.
```

Comment:
Depending on the value of `{product-title}`, I think you may have let an `OpenShift` slip through.
```suggestion
A platform Operator is an OLM-based Operator that can be installed during or after an {product-title} cluster's Day 0 operations and participates in the cluster's lifecycle. As a cluster administrator, you can use platform Operators to further customize your {product-title} Container Platform installation to meet your requirements and use cases.
```

Reply:
Ooh yes, will switch it (although in this case since it's specifically the OCP release notes, wouldn't have appeared in a different distro). TY!